apps/ligo-virgo/src/pages/workspace/docs/components/tabs/Tabs.tsx

```diff
@@ -13,9 +13,10 @@ const StyledTabs = styled('div')(({ theme }) => {
     };
 });
 
-const StyledTabTitle = styled('div', {
-    shouldForwardProp: (prop: string) => !['isActive'].includes(prop),
```

Comment:
Why was it deleted? Just to understand

Reply:
Just see https://github.com/toeverything/AFFiNE/blob/develop/libs/components/ui/src/styled/index.ts#L69-L72

We have handled `shouldForwardProp` by default